Source/WebCore/platform/PlatformKeyboardEvent.h

```diff
@@ -160,6 +162,7 @@ namespace WebCore {
         String m_code;
         String m_keyIdentifier;
         int m_windowsVirtualKeyCode { 0 };
+        int m_unmodifiedWindowsVirtualKeyCode { 0 };
```

Comment:
This doesn't tell us how "unmodified" this value is. Given the only difference is from whether input method overrode the key-code to be 229 or not, might be more straight forward to add a boolean indicating whether input-method-override is in effect or not, and then return 229 in `windowsVirtualKeyCode()` when that boolean flag is set. This will avoid having to duplicate this key-code information twice.